### ISSUE TITLE
toil-rnaseq on toil master needs mesos.interface

### ISSUE THREAD

**Issue by Jeltje** [BD2KGenomics/cgl-docker-lib#122]
I created a toil cluster, logged on to the toil-master, and ran

```
virtualenv toil
source toil/bin/activate
pip install toil-scripts
toil-rnaseq run aws:us-west-2:jeltje-jobstore-bucket --batchSystem=mesos --mesosMaster mesos-master:5050 --manifest my-manifest.tsv
```

This dies with

```
Traceback (most recent call last):
  File "/home/mesosbox/toil/bin/toil-rnaseq", line 11, in <module>
    sys.exit(main())
  File "/home/mesosbox/toil/local/lib/python2.7/site-packages/toil_scripts/rnaseq_cgl/rnaseq_cgl_pipeline.py", line 638, in main
    Job.Runner.startToil(Job.wrapJobFn(map_job, download_sample, samples, config), args)
  File "/home/mesosbox/toil/local/lib/python2.7/site-packages/toil/job.py", line 422, in startToil
    with setupToil(options, userScript=job.getUserScript()) as (config, batchSystem, jobStore):
  File "/usr/lib/python2.7/contextlib.py", line 17, in __enter__
    return self.gen.next()
  File "/home/mesosbox/toil/local/lib/python2.7/site-packages/toil/common.py", line 450, in setupToil
    batchSystemClass, kwargs = loadBatchSystemClass(config)
  File "/home/mesosbox/toil/local/lib/python2.7/site-packages/toil/common.py", line 375, in loadBatchSystemClass
    from toil.batchSystems.mesos.batchSystem import MesosBatchSystem
  File "/home/mesosbox/toil/local/lib/python2.7/site-packages/toil/batchSystems/mesos/batchSystem.py", line 25, in <module>
    import mesos.interface
ImportError: No module named mesos.interface
```

`pip list`

```
bd2k-python-lib (1.10.dev6)
boto (2.38.0)
pip (8.1.2)
PyYAML (3.11)
setuptools (21.2.2)
toil (3.1.4)
toil-scripts (2.0a2.dev177)
tqdm (3.8.0)
wheel (0.29.0)
```

Where and how do I get mesos.interface?

**Comment by jvivian** [comment]
This isn't an issue with the RNA-seq Docker container, but an issue with how the Toil-master is being setup via CGCloud.

**Comment by hannes-ucsc** [comment]
The toil-scripts virtualenv needs to be created with `--system-site-packages`. @jvivian can you make sure this is sufficiently documented?

**Comment by jvivian** [comment]
Yes — currently documentation doesn't tell users to use a virtualenv which I'll change.

I also told Jeltje once we have a stable release for toil-scripts, I can add it as part of the toil-box image.

**Comment by jvivian** [comment]
@hannes-ucsc — Can you close if your fix for the venv was merged?

**Comment by hannes-ucsc** [comment]
The fix for BD2KGenomics/toil#929 is present in Toil 3.1.6 and the latest master. We have to release a stable version of toil_scripts that refers to that version of Toil or resolves #292 before we can close this ticket. I'd prefer the latter.

**Comment by jvivian** [comment]
2.0.0 released
